mentors/kotlin/s.json

```diff
@@ -28,7 +28,7 @@
         "name": "Supriya Srivatsa",
         "link_text": "Twitter",
         "link_url": "https://twitter.com/SupriyaSrivatsa",
-        "avatar_url": "https://twitter.com/SupriyaSrivatsa/photo",
+        "avatar_url": null,
```

Comment:
This url looks like it will work better than the previous url you had:

https://pbs.twimg.com/profile_images/1173929123787370496/g8xDKkaQ_400x400.jpg

Reply:
Hey @Stargator thanks! `null` (the github avatar) works :)

Reply:
Hey @Stargator , could you look into this again and see if this can be approved please. Thank you!